src/client/containers/Layout/index.tsx

```diff
@@ -171,14 +185,17 @@ export const Layout: FC = ({ children }) => {
       <Content collapsedSidebar={collapsedSidebar} useTabbar={isMobile}>
         <Navbar
           title={t(`navigation.${path}`)}
+          subTitle={vaultName}
           walletAddress={selectedAddress}
           addressEnsName={addressEnsName}
           onWalletClick={() => dispatch(WalletActions.walletSelect({ network: currentNetwork }))}
           selectedNetwork={currentNetwork}
           networkOptions={SUPPORTED_NETWORKS}
           onNetworkChange={(network) => dispatch(NetworkActions.changeNetwork({ network: network as Network }))}
         />
+
         {children}
+
```

Comment:
These new lines are probably not needed

Reply:
Not needed perse, just more vertical whitespacing. I like it to diferentiate things and readability:
<img width="580" alt="image" src="https://user-images.githubusercontent.com/78362532/163131233-fcfb015e-b994-423a-b183-96542701ad56.png">
OR
<img width="431" alt="image" src="https://user-images.githubusercontent.com/78362532/163131276-2d2c365c-f8b0-48a8-914d-7ee5ff31ce18.png">

Reply:
I still think it's readable enough without the new lines, but up to you. If you want to leave those, I think it's not the end of the world :)